docs/source/user_guide/getting_started_with_dpf_server.rst

```diff
@@ -0,0 +1,133 @@
+.. _ref_getting_started_with_dpf_server:
+
+===============================
+Getting Started with DPF Server
+===============================
+
+What is DPF Server
+------------------
+
+The Data Processing Framework (DPF) provides numerical simulation users and engineers with a toolbox for accessing and transforming 
+simulation data. With DPF, you can perform complex preprocessing or postprocessing of large amounts of simulation data within a 
+simulation workflow.
+
+DPF Server is a package that contains all the necessary files to run the DPF Server, enabling DPF capabilities. It is available 
+on the Ansys Customer Portal. DPF Server first available version is 6.0 (2023 R2).
+
+For more information about DPF and its use, see :ref:`ref_user_guide`. 
+
+The following section details how to use DPF Server package. For a quick start with DPF Server, see :ref:`ref_getting_started`. 
+
+Installing DPF Server
+---------------------
+
+.. _target_installing_server:
+
+#. Download the ansys_dpf_server_win_v<version_to_set>.zip or ansys_dpf_server_lin_v<version_to_set>.zip file as appropriate.
+#. Unzip the package.
+#. Change to the root folder (ansys_dpf_server_win_v<version_to_set>) of the unzipped package. 
+#. In a Python environment, run the following command:
+
+.. code::
+
+    pip install -e . 
+	
+PyDPF-Core and PyDPF-Post python modules can now be used. The instructions to install and get started with PyDPF-Core 
+(ansys-dpf-core module) can be found at `Getting Started section <https://dpf.docs.pyansys.com/getting_started/install.html>`_. 
```

Comment:
Need to be updated looking at this PR: https://github.com/pyansys/pydpf-core/pull/642

Reply:
Should be ok